nautobot/dcim/views.py

```diff
@@ -360,10 +360,8 @@ class RackGroupBulkDeleteView(generic.BulkDeleteView):
 
 
 class RackListView(generic.ObjectListView):
-    queryset = (
-        Rack.objects.select_related("location", "rack_group", "tenant", "role", "status")
-        .prefetch_related("devices__device_type")
-        .annotate(device_count=count_related(Device, "rack"))
+    queryset = Rack.objects.prefetch_related("devices__device_type").annotate(
```

Comment:
Why are we prefetching device_type here?

Reply:
I'm guessing this was intended to relate to the rack utilization calculation via `Rack.get_available_units()` which uses both `Rack.devices` and `Rack.devices.device_type`. However, I see those queries being evaluated for each rack regardless of whether this `prefetch_related` is present or not, so I **think** it's not needed.